libs/shared/feature/messaging/src/lib/components/messages/messages.component.html

```diff
@@ -1,3 +1 @@
-<header class="header">
-  <h2 class="header-title">Messages</h2>
-</header>
+<div>messages works!</div>
```

Comment:
Lets match what we are doing in other places

```suggestion
<div>Under Construction</div>
```